src/firefly/js/ui/dynamic/EmbeddedPositionSearchPanel.jsx

```diff
@@ -155,34 +166,151 @@ export function EmbeddedPositionSearchPanel({
         ? <WrapperComponent {...additionalProps}>{internals}</WrapperComponent>
         : internals;
 
+    const keyVal = (k, v, isLast) => (
+        <>
+            <Typography component='span' color={'primary'}>{k}: </Typography> {v}
+            {!isLast && ', '}
+        </>
+    );
+
+    function SearchSummary({title, searchType, coords, radius, fileName, rows, showCoords, showRadius, showFileInfo, sx}) {
+
+        //Label/Key & Value pairs do display, calculating here to determine easily where the last comma should be
+        const keyValuePairs = [
+            { k: 'Search Type', v: searchType },
+            ...(showRadius && searchType === 'Cone' ? [{ k: 'Search Radius', v: radius }] : []),
+            ...(showCoords && searchType !== 'Multi-Object' && coords ? [{ k: 'Coordinates', v: coords }] : []),
+            ...(showFileInfo && searchType === 'Multi-Object' && fileName && rows ? [
+                { k: 'File Name', v: fileName },
+                { k: 'Rows', v: rows }
+            ] : [])
+        ];
+
+        return (
+            <Stack sx={sx}>
+                {title && <><Typography color={'warning'} level='title'>{title}</Typography></>}
+                <Typography color={'neutral'} level='body-md'>
+                    {keyValuePairs.map((pair, index) =>
+                        keyVal(pair.k, pair.v, index === keyValuePairs.length - 1)
+                    )}
+                </Typography>
+            </Stack>
+        );
+    }
+
+    const Header = function(isOpen) {
+        const {groupKey} = useContext(FieldGroupCtx);
+        const reqObj = getFieldGroupResults(groupKey,true);
```

Comment:
@loitly I think I get what you were trying to say earlier - instead of sending 
`searchType, coords, radius, fileName, rows` to `SearchSummary` component, I could just send/use this request object directly in `SearchSummary`? is that right?